src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -536,10 +536,18 @@ def _stream_engine_outputs(self, engine_outputs, prompts, kwargs):
                 finished_reason[0],
                 logits,
             )
-            yield TextGenerationOutput(
+            # Add session_id to schema if it exists
+            #  more relevant for `ChatPipeline`
+            schema_kwargs = (
+                {"session_ids": session_ids}
+                if (session_ids := kwargs.get("session_ids"))
```

Comment:
`:=` isn't supported in python 3.8 right?

Reply:
It is Ben